### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ayushishu**
Add objective and GAME DESCRIPTION:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,29 @@
-# tictactoe
+# TIC-TAC-TOE-GAME
 
-A new Flutter project.
+A Flutter project.
 
-## Getting Started
+## OBJECTIVE:
+This project for Data Day Grind 3.0:  with a aims to develop a Tic Tac Toe game using Flutter. It mainly consists of developing and implementing a computer program that plays Tic Tac Toe against another player.
```
Comment (hasnainmakada-99) [inline README.md]: The computer doesn't play with the human, change it to "you can play this game with your friends"

Comment (hasnainmakada-99) [reply README.md]: @ayushishu notify me for re-review once you've updated it

**Comment by ayushishu** [comment]
@hasnainmakada-99 i update the readme file once again (corrected some mistake)

**Comment by hasnainmakada-99** [comment]
> @hasnainmakada-99 i update the readme file once again (corrected some mistake)

Ok let me check it

**Review by hasnainmakada-99 — APPROVED** [review]

**Pull request merged**